spring-ai-docs/src/main/antora/modules/ROOT/pages/api/vectordbs/apache-cassandra.adoc

```diff
@@ -0,0 +1,176 @@
+= Apache Cassandra
```

Comment:
added first.

up until weaviate they are alphabetically sorted.  i'd recommend that they all are.  want me to do that ?

Reply:
Hi Michael, 

Lets not change the order now as it will require changing it in multiple places and also reordering other components for consistency.

But please raise a new issue about it. Maybe we will have time to look at it before the GA.